gulpfile.babel.ts

```diff
@@ -397,6 +399,14 @@ if (args.remote) {
         },
         import: './src/inboxsdk-js/inboxsdk-NONREMOTE',
       },
+      firefox: {
+        library: {
+          export: 'default',
+          name: 'InboxSDK',
+          type: OutputLibraryType.UMD,
+        },
+        import: './src/inboxsdk-js/inboxsdk-FIREFOX',
+      },
```

Comment:
```suggestion
```

I would prefer that we didn't add a new entry point for Firefox as our package is currently quite large.

Instead, it would be on Firefox extension developers to include both `@inboxsdk/core/firefox.js` and `@inboxsdk/core/inboxsdk.js`.

Reply:
Requiring an application to import something of ours for side effects feels gross.

I do agree that building firefox.js with webpack like this would create a lot of duplicate content in our package as it and our main entry point would contain a lot of the same bundle, and our build time would increase a bit from that.

We can support allowing applications to just import `import InboxSDK from '@inboxsdk/core/firefox';` without adding a new webpack entry point by not building firefox.js with webpack. (And in the future maybe we won't use webpack at all for any parts and deliver everything unbundled, not just firefox.js.)